vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -15,12 +15,14 @@ export class DotnetAcquisitionStarted extends IEvent {
     public readonly eventName = 'DotnetAcquisitionStarted';
     public readonly type = EventType.DotnetAcquisitionStart;
 
-    constructor(public readonly installKey: string, public readonly requestingExtensionId = '') {
+    constructor(public readonly installKey: string, public readonly startingVersion: string, public readonly requestingExtensionId = '') {
         super();
     }
 
     public getProperties() {
-        return {AcquisitionStartInstallKey : this.installKey, extensionId : TelemetryUtilities.HashData(this.requestingExtensionId)};
+        return {AcquisitionInstallKey : this.installKey,
+                AcquisitionStartVersion : this.installKey,
```

Comment:
i think this was meant to be `startingVersion`

Reply:
Good catch, ty ;)